Monika After Story/game/script-topics.rpy

```diff
@@ -3662,7 +3721,8 @@ label monika_herself:
     m 2eua "If nothing else, this can be our comfy little shelter from the cruelty of the outside world."
     m 1eka "Come talk to me about your problems if it gets too much for you to handle."
     show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve
```

Comment:
You have this dissolve before an if block, so it's possible that you would just dissolve for no reason and return.